Adds index files to the new Payment Services Extension folder and subfolders in GraphQL docs (#390)
AdobeDocs/commerce-webapi: src/pages/graphql/payment-services-extension/mutations/index.md

---
title: Payment Services extension mutations
description: Learn how to use the GraphQL mutations to integrate the Payment Services extension.
keywords:
  - GraphQL
  - Payments
---

# Payment Services extension mutations

> The Payment Services extension mutations allow you to perform operations such as creating, updating, and deleting payment-related data, enabling you to manage the payment services extension with ease.

Reviewer: ```suggestion
The mutations described in this section allow you to create, update, and delete payment-related data with the Payment Services extension.
```
